packages/dashboard/src/customization/widgets/kpi/icon.tsx

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
-
+import React, { useState } from 'react';
 import { default as KPISvg } from './KPI.svg';
 import { default as KPISvgDark } from './KPI-dark.svg';
 
-const KPIIcon: React.FC = () => (
-  <span>
-    <img src={KPISvg} alt='KIP widget light icon' />
-    <img src={KPISvgDark} alt='KPI widget dark icon' />
-  </span>
-);
+const KPIIcon: React.FC = () => {
+  const [over, setOver] = useState(false);
+  return (
+    <span onMouseOver={() => setOver(true)} onMouseOut={() => setOver(false)}>
```

Comment:
its probabaly worth creating a WidgetIcon component where you pass the svgs as a props 
example:
<WidgetIcon defaultIcon={} darkIcon={} >

Reply:
Resolved